types/graphql-upload/processRequest.d.mts

```diff
@@ -1,19 +1,52 @@
-import { IncomingMessage, ServerResponse } from "http";
+export default function processRequest(
+    request: import("node:http").IncomingMessage,
+    response: import("node:http").ServerResponse,
+    options?: ProcessRequestOptions,
+): Promise<
+    Array<
+        {
+            [key: string]: unknown;
+        } | {
+            [key: string]: unknown;
+        }
```

Comment:
Same type twice in a union?

Reply:
My bad while fixing the style. This should be `{ [key: string]: unknown } | Array<{ [key: string]: unknown }>`. Fixed.